src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,122 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StorageDaoImplTest {
+    private static RegistrationService registrationServise;
+    private static User bob;
+    private static User alise;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationServise = new RegistrationServiceImpl();
+        bob = new User();
+        alise = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        bob.setId(11L);
+        bob.setLogin("Volodymyr");
+        bob.setPassword("password");
+        bob.setAge(21);
+        alise.setId(12L);
+        alise.setLogin("1abcdef");
+        alise.setPassword("password");
+        alise.setAge(20);
+    }
```

Comment:
Bob with login "Volodymyr" can be a bit confusing, don't you find?)
Could use more generic variable names (like user1, user2 etc), or logins that match these variables, like "Bob_nagibator" for User bob)